pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -96,15 +96,24 @@ type windows struct {
 }
 
 // New returns a new Windows instance constructed from the given WindowsVM
-func New(ipAddress, instanceID, workerIgnitionEndpoint, vxlanPort string, signer ssh.Signer) (Windows, error) {
+func New(ipAddress, providerName, instanceID, workerIgnitionEndpoint, vxlanPort string, signer ssh.Signer) (Windows, error) {
 	if workerIgnitionEndpoint == "" {
 		return nil, errors.New("cannot use empty ignition endpoint")
 	}
 
+	var adminUser string
+	if providerName == "azure" {
+		adminUser = "capi"
```

Comment:
How did you find this out? :-)

Reply:
I went through the Azure machine controller code to find out what's happening, and the user name (from a const) is passed here: https://github.com/openshift/cluster-api-provider-azure/blob/master/pkg/cloud/azure/services/virtualmachines/virtualmachines.go#L122

Reply:
Please add a comment with a  link to whatever story @ravisantoshgudimetla or you create to normalize this.